include/ssp_var_defs.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * variable definitions to enable bootmgr self revocation
+ */
+#ifndef SSP_VAR_DEFS_H_
+#define SSP_VAR_DEFS_H_
+
+#define SSPVER_SIZE 8
+#define SSPSIG_SIZE 131
```

Comment:
Those sizes came from the spec, I don't want to accidentally set something that doesn't match here since I have to assume that the consumer may make some explicit assumptions about the length as well.

Reply:
I think I came up with the right solution here. I kept the size definitions and assert the actual size of the structures at build time. I should probably put the same thing into the revocations mule.